src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="#"
+      method="post"
+      class="form-field"
+      data-qa="big"
+    >
+      <label class = "bigInput">
+        <img 
+          src="./images/Search.svg"
+          alt="search" 
+          class="icon"
+        >
+        <input 
+          type="text"
+          data-qa="keypress"
+          placeholder="Try &quot;Los Angeles&quot;"
+          class="form_input"
+        >
+
+      </label>
+    </form>
+
+    <form 
+      action="#"
+      method="post"
+      class="form-field-small"
+      data-qa="small"
+    >
+      <label class = "bigInput">
+        <img 
+          src="./images/Search.svg"
+          alt="search" 
+          class="iconSmall"
+        >
+        <input 
+          type="text"
+          data-qa="keypress"
+          placeholder="Try &quot;Los Angeles&quot;"
+          class="form_input--small"
```

Comment:
Do not use a class modifier without the class itself
```suggestion
          class="form_input-small"
```